src/routers/AlertsRouter.js

```diff
@@ -3,6 +3,10 @@ import { AppDevRouter } from 'appdev';
 import type Request from 'express';
 import AlertsUtils from '../utils/AlertsUtils';
 
+/**
+ * Express Router object for retrieving TCAT alert data
```

Comment:
```suggestion
 * Router object for retrieving TCAT alert data
```
Can you explain more what a "TCAT alert" is? I'm personally unsure of what it is, so I'm sure other people have the same question!